backend/services/interfaces/testSessionService.ts

```diff
@@ -127,10 +127,11 @@ export interface ITestSessionService {
   /**
    * delete a TestSession with the given id, return deleted id
    * @param id id to delete
+   * @param now optional date to use as the current date
    * @returns deleted id
-   * @throws Error if deletion fails
+   * @throws Error if deletion fails or given test session has already started
    */
-  deleteTestSession(id: string): Promise<string>;
+  deleteTestSession(id: string, now?: Date): Promise<string>;
```

Comment:
i'm thinking we don't need to pass the date from the frontend bc i don't imagine they'll be trying to delete RIGHT before it starts and even if they do, its pretty hard for a student to enter access code + select name + click start all before deletion goes through